main.go

```diff
@@ -118,15 +119,20 @@ func main() {
 		log.Fatalf("⚠ Failed to create GSuite Groupssettings API client: %v", err)
 	}
 
-	licSrv, err := glib.NewLicensingService(clientSecretFile, impersonatedUserEmail)
+	licnsSrv, err := glib.NewLicensingService(clientSecretFile, impersonatedUserEmail)
 	if err != nil {
 		log.Fatalf("⚠ Failed to create GSuite Licensing API client: %v", err)
 	}
 
+	licSrv := glib.LicensingService{
+		Service:          licnsSrv,
+		ThrottleRequests: throttleRequests,
+	}
+
```

Comment:
What made you wrap the licensing service here instead of returning it directly from the NewLicensingService() function?